kythe/cxx/indexer/cxx/KytheVFS.h

```diff
@@ -32,9 +33,17 @@ class IndexVFS : public llvm::vfs::FileSystem {
  public:
   /// \param working_directory The absolute path to the working directory.
   /// \param virtual_files Files to map.
+  /// \param virtual_dirs Directories to map.
+  /// \param style Style used to parse incoming paths. Paths are normalized
+  /// to POSIX-style.
   IndexVFS(const std::string& working_directory,
            const std::vector<proto::FileData>& virtual_files,
-           const std::vector<llvm::StringRef>& virtual_dirs);
+           const std::vector<llvm::StringRef>& virtual_dirs,
+           const llvm::sys::path::Style style);
```

Comment:
top-level `const` is meaningless clutter on value parameters in function declarations, please omit.